### DISCUSSION TITLE
Tooltip for help info

### DISCUSSION THREAD

**Discussion by segalion** [home-assistant/frontend#11187] [Other feature requests]
HA and Lovelace are so great that can be used in semi-professional enviroment. People could create automation systems for others, and could be great to has the hability to support a HELP tooltip that explains the meaning of the entity (i.e. a input_number to change some internal parameter of the system).
With this, when someone hover on a entity for 2 seconds, if it has defined a "help" or "tooltip" parameter, a tooltip will be showed with info associated to the parameter. 

Could be great even jinja to make tooltip dynamic (i.e. showing text and value, calcs, etc.)

I dont know how the mobile view can have "hover". Maybe it dont have sense, because mobile is a "simplified view" of desktop ui.

**Comment by iantrich** [comment]
One solution for mobile would be to select "Help" from a dropdown which puts you in a mode where anything you click would show the tooltip. That being said, this would be a big change to tackle

**Comment by bramkragten** [comment]
I think this should be an option for the Lovelace card/row/element
